src/server/container_utils.cc

```diff
@@ -337,9 +337,10 @@ OpResult<string> RunCbOnFirstNonEmptyBlocking(Transaction* trans, int req_obj_ty
   }
 
   auto wcb = [](Transaction* t, EngineShard* shard) { return t->GetShardArgs(shard->shard_id()); };
-  const auto key_checker = [req_obj_type](EngineShard* owner, const DbContext& context,
-                                          Transaction*, std::string_view key) -> bool {
-    return owner->db_slice().FindReadOnly(context, key, req_obj_type).ok();
+  auto ns = &trans->GetNamespace();
+  const auto key_checker = [req_obj_type, ns](EngineShard* owner, const DbContext& context,
```

Comment:
seems you do not use `ns` ?

Reply:
See line 343 below: `ns->GetDbSlice()`...